scienzati_bot.py

```diff
@@ -4,14 +4,15 @@
 import sqlite3
 import random
 import re
+import sys
 
 #Settings class
 
 class Settings:
 	TelegramApiKey = "676490981:AAELlmTlQLD4_1HojhzWIX4yISDrVU5qDmA"
```

Comment:
API Key to change from hardcoded to inputed and then change the current APIkey in by bot father.

Reply:
Fixed with last commit